setup.py

```diff
@@ -60,14 +60,12 @@
     # Please keep alphabetized
     'baselines @ https://api.github.com/repos/openai/baselines/tarball/f2729693253c0ef4d4086231d36e0a4307ec1cb3',  # noqa: E501
     'flake8',
-    'flake8-docstrings==1.3.0',
+    'flake8-docstrings==1.4.0',
     'flake8-import-order',
     'pandas',
     'pep8-naming==0.7.0',
     'pre-commit',
-    # pydocstyle 4.0.0 breaks flake8-docstrings 1.3.0
-    # See https://gitlab.com/pycqa/flake8-docstrings/issues/36
-    'pydocstyle<4.0.0',
+    'pydocstyle<4.1,>=4.0',
```

Comment:
Shouldn't this be `<4.1,>4.0.0`?

Reply:
The updated flake8-docstrings works fine with pydocstyle 4.0.0, so `>=` should be fine

Reply:
Oh okay I see what I did there. The PR says I updated pydocstyle to 4.0.1 while I actually set it to >=4.0.0 before submitting. 4.0.0 works fine though, so I'll update the PR description.